assets/js/blocks/cart-checkout/checkout-i2/form-step/additional-fields.tsx

```diff
@@ -2,27 +2,19 @@
  * External dependencies
  */
 import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
-import {
-	RegisteredBlocks,
-	getRegisteredBlockTemplate,
-} from '@woocommerce/blocks-checkout';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
 import { useForcedLayout } from '../use-forced-layout';
+import { getRegisteredBlockNamesByParent } from '../editor-utils';
 
-export const AdditionalFields = ( {
-	area,
-}: {
-	area: keyof RegisteredBlocks;
-} ): JSX.Element => {
-	const registeredBlocks = getRegisteredBlockTemplate( area );
+export const AdditionalFields = ( { area }: { area: string } ): JSX.Element => {
```

Comment:
Might be worth renaming area to blockName to keep consistency.

Reply:
Changed to block in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4684/commits/6a879e327188f968fac8c314d1d0f7ccfd566b86